client/landing/stepper/declarative-flow/internals/steps-repository/migration-trial/trial-plan.tsx

```diff
@@ -0,0 +1,91 @@
+import { getPlan, PLAN_BUSINESS, PLAN_WPCOM_PRO } from '@automattic/calypso-products';
+import { SiteDetails } from '@automattic/data-stores';
+import { localizeUrl } from '@automattic/i18n-utils';
+import { Title, SubTitle, NextButton } from '@automattic/onboarding';
+import { createElement, createInterpolateElement } from '@wordpress/element';
+import { sprintf } from '@wordpress/i18n';
+import { useI18n } from '@wordpress/react-i18n';
+import React, { useState } from 'react';
+import { isEligibleForProPlan } from 'calypso/my-sites/plans-comparison';
+import { useSelector } from 'calypso/state';
+import useUnsupportedTrialFeatureList from './hooks/use-unsupported-trial-feature-list';
+import TrialPlanFeaturesModal from './trial-plan-features-modal';
+
+interface Props {
+	site: SiteDetails;
+}
+const TrialPlan = function TrialPlan( props: Props ) {
+	const { __ } = useI18n();
+	const { site } = props;
+	const [ showPlanFeaturesModal, setShowPlanFeaturesModal ] = useState( false );
+
+	const targetSiteEligibleForProPlan = useSelector( ( state ) =>
+		isEligibleForProPlan( state, site?.ID )
+	);
```

Comment:
It's up to you, but I don't think we need to include any of this Pro plan logic. There's still a feature flag for it, but we've basically abandoned the Pro plan for new sites as far as I can tell.

Reply:
Thanks for the feedback. The PRO plan conditional logic has been removed!